### PR TITLE
Azure Communication Services Chat (Certified Connector)

### PR DESCRIPTION
**Pull request by lei-msft**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Azure Communication Services Chat/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Create a chat given a chat topic. 
+
+### List Chat Threads
+List all chat threads.
+
+### List Thread Messages
+List Messages From Thread.
+
+### Get Thread Properties
+Get the properties of a chat thread.
```
[the remark this answers is not in the document]
Comment (ikumarapeli-msft) [reply certified-connectors/Azure Communication Services Chat/README.md:46]: ill create a pr to update the original repo as well

**Review by ikumarapeli-msft — DISMISSED** [review]

File: certified-connectors/Azure Communication Services Chat/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,725 @@
...
+          }
+        }
+      ]
+    }
+  },
+    "/chat/threads/{chatThreadId}/participants/%3Aadd": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Success."
```
Comment (v-EgorKozhadei) [inline certified-connectors/Azure Communication Services Chat/apiDefinition.swagger.json:221]: @lei-msft can you please provide a schema for 2xx response?

File: certified-connectors/Azure Communication Services Chat/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,725 @@
...
+                  },
+                  "description": "Participants to add."
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/chat/threads/{chatThreadId}/participants/%3Aremove": {
```
Comment (v-EgorKozhadei) [inline certified-connectors/Azure Communication Services Chat/apiDefinition.swagger.json:279]: @lei-msft please check the encoding here too.

File: certified-connectors/Azure Communication Services Chat/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,725 @@
...
+      "post": {
+        "responses": {
+          "204": {
+            "description": "Success."
```
Comment (v-EgorKozhadei) [inline certified-connectors/Azure Communication Services Chat/apiDefinition.swagger.json:283]: @lei-msft can you please provide schemas for all 2xx responses?

File: certified-connectors/Azure Communication Services Chat/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+          "tooltip": "Find your endpoint URL in your Azure Communication Services resource in Azure Portal in the \"Overview\" tab.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Azure Communication Services Chat/apiProperties.json:18]: @lei-msft default icon color is [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), can you please change it?

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @lei-msft! Please check comments after the review, thank you!

**Comment by v-EgorKozhadei** [comment]
Hello @lei-msft, 

I hope you are doing well. 

I am following up with you to see how things are going with preparing your connector. 

Please let us know if you have any questions. 

Thank you.

**Pull request merged**